www/source/docs/best-practices.html.md.erb

```diff
@@ -8,6 +8,7 @@ This chapter covers recommended best practices for runtime and buildtime.
 We focus on best practices for packages that use Habitat Builder for continuous builds.
 
 ## Table of Contents
+  - [Running Habitat Linux Containers](#running-habitat-linux-containers)
```

Comment:
Does this title include the keyword(s) you think people would be searching for? 
Specifically, would it be beneficial to work terms like root, non-root, or OpenShift into this text?

Reply:
@ryankeairns All those are contained in the text, for sure... is there some way to add additional metadata to ensure that the appropriate sections bubble up in searches of the site?

Reply:
A few things come to mind... putting these keywords in the meta description atop the parent page; manually promoting these in Swiftype search rankings; and linking to this page when we build out landing pages for topics such as 'Habitat and Docker' or 'Habitat and OpenShift'.

I can handle the latter two after this is merged (the last one will take some time for content to develop), if you can double check the description.

Thanks!

Reply:
re: the meta description, it would be here:
https://github.com/habitat-sh/habitat/blob/fc10e7e8ea657e0ce55c440ed855a26fa20d8b85/www/source/docs/best-practices.html.md.erb#L3